### PR TITLE
Weather App, by Maria Bergman and Vladimir Bykov

### PR DESCRIPTION
**Pull request by vladjnbykov**
Link to our deployed project: https://elegant-davinci-7eedac.netlify.app/

My reflections on how this week's project turned out: Great

Things I'd like to have clarified or explained in more detail: conditional display of a background image (in the body) depending on media queries.

### CODE REVIEW

**Review by Ulve17 — APPROVED** [review]
Hi Guys!! Great job with the project this week. 🌈 
Clean and nice code! 😄 
Hopefully we dindn't miss anything, significant.  😨 
Ati, Nazanin and Ulysses 😸

**Comment by vladjnbykov** [comment]
Hi guys,
Thank you very much for your comments.
We have now fixed everything and set IDs as camelCases. App works and we 
have now pushed it back to GitHub.

Have a nice day!
Maria and Vlad


------ Originalmeddelande ------
Från: "Ulve17" <notifications@github.com>
Till: "Technigo/project-weather-app" 
<project-weather-app@noreply.github.com>
Kopia: "Vladimir JN Bykov" <vlad.jnbykov@gmail.com>; "Author" 
<author@noreply.github.com>
Skickat: 2021-02-24 20:52:52
Ämne: Re: [Technigo/project-weather-app] Weather App, by Maria Bergman 
and Vladimir Bykov (#163)

File: code/app.js [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+          </tr>
+          `
+        }
+      })
+    })
+  })
+}
+weatherForecast()
+
+/*************************WEATHER GRAPH *******************/
```
Comment (KarolCislowski) [inline code/app.js:190]: Greate that you used a very nice library to create charts 🤓

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
You created a very nice working app but you need to work some more on styling. Your app does not scale well beyond mobile screen sizes. 🎨

**Comment by vladjnbykov** [comment]
Hi Karol,

We have now done some changes to the app which we hope are adressing the 
responsivity issues:

1. Background image was removed from mobile phone view (before it was 
hiding behind the widget)
2. We changed position of the widget and placed it in the center of the 
larger screen
3. We have added a footer for the desktop screen.

Thank you very much,
Best regards,

Maria and Vlad


------ Originalmeddelande ------
Från: "Karol Cislowski" <notifications@github.com>
Till: "Technigo/project-weather-app" 
<project-weather-app@noreply.github.com>
Kopia: "Vladimir JN Bykov" <vlad.jnbykov@gmail.com>; "Author" 
<author@noreply.github.com>
Skickat: 2021-02-25 17:22:29
Ämne: Re: [Technigo/project-weather-app] Weather App, by Maria Bergman 
and Vladimir Bykov (#163)

**Review by KarolCislowski — APPROVED** [review]
Thank you for a fast response 👍🏻 Now it is much better 💯

**Pull request closed without merging**